src/modules/stream/internal.js

```diff
@@ -73,6 +75,36 @@ async function handleYoutubeStream(streamInfo, res) {
     }
 }
 
+function transformHLSMediaPlaylist(streamInfo, hlsPlaylist) {
+    function generateInternalStreamsOfSegments(segment) {
```

Comment:
convert this local function to a const arrow function and rename it to something shorter and more readable, such as `makeInternalSegments`